v2/types.go

```diff
@@ -35,7 +35,7 @@ type Service struct {
 	// a GET on the binding resource's endpoint
 	// (/v2/service_instances/instance-id/service_bindings/binding-id) is
 	// supported for all plans.
-	BindingRetrievable bool `json:"binding_retrievable,omitempty"`
+	BindingRetrievable bool `json:"bindings_retrievable,omitempty"`
```

Comment:
The name of this field should change so that it matches the schema - so call it `BindingsRetrievable`.